### DISCUSSION TITLE
No available runners for `macos-latest-xl`?

### DISCUSSION THREAD

**Discussion by MetRonnie** [actions/virtual-environments#7526] [General questions]
Is this a known issue at the moment? Was hoping to test out the new faster runners but am not able to

> This request was automatically failed because there were no enabled runners online to process the request for more than 1 days.

https://github.com/MetRonnie/cylc-flow/actions/runs/4860633659/attempts/1

**Comment by lschmierer** [comment]
I encountered this as well. I am actually not quite sure if the *-xl runners are actually available for free for open-source projects.

**Reply by MetRonnie** [reply]
Ah maybe it's a documentation problem then

**Reply by shamil-mubarakshin** [reply]
Hello @MetRonnie.
*-xl runners are 12-core macOS runners, charged according to [per-minute rates](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#per-minute-rates)
